parsl/tests/manual_tests/test_ad_hoc_htex.py

```diff
@@ -0,0 +1,49 @@
+import parsl
```

Comment:
It would be better to put this config in `parsl/tests/configs/` and check that `pytest` can run against it. (I have verified that pytest runs against my own adhoc cluster config... so this should be fine).

Even better would be to parameterize all of `remotes`, `worker_logdir_root`, `address`, `worker_init` `username` `script_dir` into user_opts.py - `address` already has a `user_opts` key of `public_ip`.

Reply:
I tidied up my config into this branch https://github.com/Parsl/parsl/tree/benc-adhoc-test-conf which you could merge if useful.

Reply:
Just merged it.